.gitpod.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+
+# Install Julia
+RUN sudo apt-get update \
+    && sudo apt-get install -y \
+        build-essential \
+        libatomic1 \
+        python \
+        gfortran \
+        perl \
+        wget \
+        m4 \
+        cmake \
```

Comment:
Note:  `cmake` is already installed in `gitpod/workspace-full`:

https://github.com/gitpod-io/workspace-images/blob/b17e2ea52123f14c26ddb99e0a9f2f233312bf25/full/Dockerfile#L117-L118

Reply:
Done!